test/integration/test_resolvers.py

```diff
@@ -0,0 +1,24 @@
+"""Integration tests for the conda dependency resolution."""
+
+import os
+from base import integration_util
+from base.api import ApiTestCase
+
+GNUPLOT = {u'version': u'4.6', u'type': u'package', u'name': u'gnuplot'}
+
+class CondaResolutionIntegrationTestCase(integration_util.IntegrationTestCase, ApiTestCase):
```

Comment:
Or downgrading to svgwrite==1.1.3, which makes me think that svgwrite has a path problem, considering the [changelog](https://pypi.python.org/pypi/svgwrite)